src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            String userPassword = user.getPassword();
+            if (userPassword.equals(password)) {
+                return true;
+            }
+        }
         return false;
```

Comment:
Let's shorten this code. You can implement this method without any if-blocks. Try to use a ternary operator